docs/api.rst

```diff
@@ -21,9 +21,20 @@ Constrained
 
     jaxopt.ProjectedGradient
     jaxopt.MirrorDescent
-    jaxopt.QuadraticProgramming
     jaxopt.ScipyBoundedMinimize
 
+Quadratic Programming
+~~~~~~~~~~~~~~~~~~~~~
+
+.. autosummary::
+  :toctree: _autosummary
+
+    jaxopt.EqualityConstrainedQP
+    jaxopt.CvxpyQP
+    jaxopt.BoxOSQP
+    jaxopt.OSQP
+    jaxopt.QuadraticProgramming
```

Comment:
I guess we can remove this one from the documentation (we just keep the code until v0.3 in case someone was using it)